_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -228,10 +290,22 @@ This element is neither an `img` element nor has a role of `img`.
 <div aria-label="W3C logo"></div>
 ```
 
+#### Inapplicable Example 5
+
+This `img` element has no `src` attribute so does not render an image.
+
+```html
+<img />
+```
+
```

Comment:
I think we need Inapplicable Examples for the other cases of not rendering an image:
* image not fully loaded (see https://act-rules.github.io/rules/e88epe#inapplicable-example-10) OTOH, we start having a few of these springled in various rules, so maybe we don't want to duplicate too much of the test (@WilcoFiers WDYT?)
* CSS `background-image` that doesn't draw anything  (I don't even know how to do that…)

Reply:
I've added 2 new Inapplicable Examples. I've also referenced the HTML definition of 'represents nothing' in the examples without `alt` and `src` since I remember one of the 508 people F2F saying they thought this should fail.